pillar/heroku/discussions.sls

```diff
@@ -91,7 +91,8 @@
       'SOCIAL_AUTH_MICROMASTERS_LOGIN_URL': 'https://micromasters.mit.edu/login/edxorg/?next=/discussions/',
       'SOCIAL_AUTH_SAML_SP_ENTITY_ID': 'https://discussions.odl.mit.edu/saml/metadata',
       'TIKA_SERVER_ENDPOINT': 'https://tika-production-apps.odl.mit.edu',
-      'vault_env_path': 'production-apps'
+      'vault_env_path': 'production-apps',
+      'OCW_NEXT_URL': 'ocw-beta.odl.mit.edu'
```

Comment:
The variable that I learned about recently for governing CORS access was `OPEN_DISCUSSIONS_CORS_ORIGIN_WHITELIST`.  Is this in lieu of that variable? Does `OPEN_DISCUSSIONS_CORS_ORIGIN_WHITELIST` need to be set, too?

Reply:
so we set this value in this dictionary here and then further on down below it's pulled out and added to the `OPEN_DISCUSSIONS_CORS_ORIGIN_WHITELIST` variable, alongside another value for Micromasters

Reply:
sorry I don't know the terminology for all this stuff because I'm not so familiar with this salt stuff

Reply:
Oh, I see. That's fine, thanks.